cmd/server/helper_keys.go

```diff
@@ -18,7 +18,7 @@ func createRS256KeysIfNotExist(c *config.Config, set, kid, use string) {
 	if _, err := ctx.KeyManager.GetKey(set, kid); errors.Cause(err) == pkg.ErrNotFound {
 		logrus.Infof("Key pair for signing %s is missing. Creating new one.", set)
 
-		keys, err := generator.Generate("")
+		keys, err := generator.Generate(set)
```

Comment:
Ok, I've been thinking on the least breaking change here, and let me run this by you. What if `getKey()` supported wildcards? Then the policy could be `mykeyset:public:29873489234`, to retrieve that key you could run `getKey("mykeyset", "public*")` which would return all public keys in that set.